hubble-be/src/main/java/com/baidu/hugegraph/filter/AuthFilter.java

```diff
@@ -51,15 +67,65 @@ public void doFilter(ServletRequest servletRequest,
                          FilterChain filterChain)
                          throws IOException, ServletException {
         try {
-            String authorization = ((HttpServletRequest) servletRequest)
-                                   .getHeader(HttpHeaders.AUTHORIZATION);
-            if (StringUtils.isNotEmpty(authorization)) {
-                this.client.setAuthContext(authorization);
+            HttpServletRequest request = (HttpServletRequest) servletRequest;
+            String authorization = request.getHeader(HttpHeaders.AUTHORIZATION);
+
+            // Missed token and request uri not in white list
+            if (StringUtils.isEmpty(authorization) && !isWhiteAPI(request)) {
+                String msg = "Missed authorization token";
+                writeResponse(servletResponse, () -> {
+                    return Response.builder()
+                                   .status(Constant.STATUS_BAD_REQUEST)
+                                   .message(msg)
+                                   .build();
+                });
+                return;
             }
+            // Illegal token format
+            if (StringUtils.isNotEmpty(authorization) &&
+                !authorization.startsWith(BEARER_TOKEN_PREFIX)) {
+                String msg = "Only HTTP Bearer authentication is supported";
+                writeResponse(servletResponse, () -> {
+                    return Response.builder()
+                                   .status(Constant.STATUS_BAD_REQUEST)
+                                   .message(msg)
+                                   .build();
+                });
+                return;
+            }
+
+            this.client.setAuthContext(authorization);
 
             filterChain.doFilter(servletRequest, servletResponse);
         } finally {
             this.client.resetAuthContext();
         }
     }
+
+    private static String buildPath(RequestMethod method, String path) {
+        return buildPath(method.name(), path);
+    }
+
+    private static String buildPath(String method, String path) {
+        return String.join(":", method, path);
+    }
+
+    private static boolean isWhiteAPI(HttpServletRequest request) {
+        String url = request.getRequestURI();
+        return WHITE_API.contains(buildPath(request.getMethod(), url));
+    }
+
+    private void writeResponse(ServletResponse servletResponse,
+                               Supplier<Response> responseSupplier) {
+        Response response = responseSupplier.get();
+
+        servletResponse.setCharacterEncoding("UTF-8");
+        servletResponse.setContentType(MediaType.APPLICATION_JSON);
+
+        try (PrintWriter writer = servletResponse.getWriter()) {
+            writer.print(JsonUtil.toJson(response));
```

Comment:
seems can call servletResponse.sendError()

Reply:
not have this api